src/glyph/BBox.js

```diff
@@ -45,20 +45,24 @@ export default class BBox {
   }
 
   addPoint(x, y) {
-    if (x < this.minX) {
-      this.minX = x;
-    }
+    if (Math.abs(x) != Infinity) {
```

Comment:
I fixed in the merge: https://github.com/devongovett/fontkit/commit/a09fb89424038dd8f8dd7d3fc298b942fc7feb63#diff-946f3492c0735c3b65fde51ae83c6cb7R48